docs/core/learn-more/authentication.md

```diff
@@ -1,35 +1,97 @@
-# Authentication
+# Authentication and Authorization
 
-## Introduction
+## Authentication
+
+Chain Core allows control of which entities have access to certain features of the system. There are two methods available for limiting access:
+
+1. Access tokens using HTTP Basic Authentication
+2. x509 client certificates
+
+These authentication objects can be created and managed via the Chain Core Dashboard, SDKs, or the [`corectl`](../reference/corectl) command line tool.
+
+For convenience, in all pre-packaged installations of Chain Core **access from localhost does not require authentication**.
```

Comment:
That's probably good enough yeah. Even if docker works with `-net=host`, I'd still call that a desktop installation.